  0%|                                                                                                                                                                                                | 0/10 [00:00<?, ?it/s]
Traceback (most recent call last):
  File "C:\Users\tobia\Documents\GitHub\RNN_Tobia_Simmler\main.py", line 15, in <module>
    run(config["experiment"])
  File "C:\Users\tobia\Documents\GitHub\RNN_Tobia_Simmler\src\runner.py", line 63, in run
    tm.fit(num_epochs=config["training"]["n_epochs"])
  File "C:\Users\tobia\Documents\GitHub\RNN_Tobia_Simmler\src\training\training_abstractbaseclass.py", line 77, in fit
    self.create_weights_histogram(cur_epoch, num_epochs)
  File "C:\Users\tobia\Documents\GitHub\RNN_Tobia_Simmler\src\training\training_abstractbaseclass.py", line 271, in create_weights_histogram
    self.save_weight_histogram(w_in, "w_in", cur_epoch + 1, num_epochs)
  File "C:\Users\tobia\Documents\GitHub\RNN_Tobia_Simmler\src\training\training_abstractbaseclass.py", line 283, in save_weight_histogram
    plt.savefig(self.output_path / "histograms" / filepath_name)
  File "C:\Users\tobia\AppData\Local\Programs\Python\Python310\lib\site-packages\matplotlib\pyplot.py", line 1023, in savefig
    res = fig.savefig(*args, **kwargs)
  File "C:\Users\tobia\AppData\Local\Programs\Python\Python310\lib\site-packages\matplotlib\figure.py", line 3343, in savefig
    self.canvas.print_figure(fname, **kwargs)
  File "C:\Users\tobia\AppData\Local\Programs\Python\Python310\lib\site-packages\matplotlib\backend_bases.py", line 2366, in print_figure
    result = print_method(
  File "C:\Users\tobia\AppData\Local\Programs\Python\Python310\lib\site-packages\matplotlib\backend_bases.py", line 2232, in <lambda>
    print_method = functools.wraps(meth)(lambda *args, **kwargs: meth(
  File "C:\Users\tobia\AppData\Local\Programs\Python\Python310\lib\site-packages\matplotlib\backends\backend_agg.py", line 509, in print_png
    self._print_pil(filename_or_obj, "png", pil_kwargs, metadata)
  File "C:\Users\tobia\AppData\Local\Programs\Python\Python310\lib\site-packages\matplotlib\backends\backend_agg.py", line 458, in _print_pil
    mpl.image.imsave(
  File "C:\Users\tobia\AppData\Local\Programs\Python\Python310\lib\site-packages\matplotlib\image.py", line 1689, in imsave
    image.save(fname, **pil_kwargs)
  File "C:\Users\tobia\AppData\Local\Programs\Python\Python310\lib\site-packages\PIL\Image.py", line 2429, in save
    fp = builtins.open(filename, "w+b")
FileNotFoundError: [Errno 2] No such file or directory: 'io\\output\\rnn1\\histograms\\histogram_w_in_1.png'
Using device: cpu
-------------------------- io\output\rnn1\histograms\histogram_w_in_1.png